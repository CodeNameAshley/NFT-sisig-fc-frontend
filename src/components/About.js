import React from 'react'

export default function About() {

  const description = "Placeholder description"

  return (
    <div>
        <h1>
            About the team
        </h1>
        <div>
          {description}
        </div>
    </div>
  )
}
