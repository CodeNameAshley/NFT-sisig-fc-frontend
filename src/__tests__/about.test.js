import { getByText, render } from "@testing-library/react";
import About from "../components/About";


it("Should show about the team on the page", () => {
  const {getByText} = render(<About />)
  const title = getByText(/About the team/)
  
  expect(title).toBeInTheDocument()
});