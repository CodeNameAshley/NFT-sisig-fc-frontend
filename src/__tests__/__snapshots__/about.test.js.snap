// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should show about the team on the page 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <h1>
      SFC
    </h1>
  </div>
</DocumentFragment>
`;

exports[`Should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <h1>
      SFC
    </h1>
  </div>
</DocumentFragment>
`;
